cmd/test_load.go

```diff
@@ -0,0 +1,199 @@
+package cmd
+
+import (
+	"archive/tar"
+	"bytes"
+	"fmt"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"go.k6.io/k6/errext"
+	"go.k6.io/k6/errext/exitcodes"
+	"go.k6.io/k6/js"
+	"go.k6.io/k6/lib"
+	"go.k6.io/k6/lib/metrics"
+	"go.k6.io/k6/loader"
+)
+
+const (
+	testTypeJS      = "js"
+	testTypeArchive = "archive"
+)
+
+type loadedTest struct {
+	testPath        string // contains the raw string the user supplied
+	source          *loader.SourceData
+	fileSystems     map[string]afero.Fs
+	runtimeOptions  lib.RuntimeOptions
+	metricsRegistry *metrics.Registry
+	builtInMetrics  *metrics.BuiltinMetrics
+	initRunner      lib.Runner // TODO: rename to something more appropriate
```

Comment:
Just `runner` I mean it seems to be the *only* runner either way.

Reply:
Maybe, but I'd prefer to wait with the final refactor and rename before messing with the names more...